src/components/link.tsx

```diff
@@ -0,0 +1,26 @@
+import Link from "next/link";
+
+interface ILink {
+  title: string;
+  href: string;
+  arrow: "back" | "forward" | "outward";
+}
```

Comment:
Let's make this property optional by adding that question mark, since there are some occasions where we may want a link without any arrow icon, for example: